SteamList/Controllers/FavoritesViewController.swift

```diff
@@ -7,23 +7,33 @@
 
 import UIKit
 
-final class FavoritesViewController: UIViewController {
+final class FavoritesViewController: UIViewController, UISearchResultsUpdating {
     
     @IBOutlet weak var tableView: UITableView!
     private let games = MockData.games
+    var searchController: UISearchController!
+    var filteredData = [Game]()
```

Comment:
it's a bad approach to put an instance of the Game inside the array as the array will never be empty. 
Better to put it this way `var filteredData: [Game] = []`